orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -374,17 +416,24 @@ func (m *swiftDialogMDMMigrator) renderMigration() error {
 				}
 			}
 
+			if err := m.mrw.SetMigrationFile(constant.MDMMigrationTypeManual); err != nil {
```

Comment:
seems like this is the only place where se call `SetMigrationFile` and it's hardcoded to manual